core/rlp_test.go

```diff
@@ -50,9 +50,8 @@ func getBlock(transactions int, uncles int, dataSize int) *types.Block {
 		address = crypto.PubkeyToAddress(key.PublicKey)
 		funds   = big.NewInt(50000 * 225000000000 * 200)
 		gspec   = &Genesis{
-			Config:   params.TestChainConfig,
-			Alloc:    GenesisAlloc{address: {Balance: funds}},
-			GasLimit: params.TestChainConfig.FeeConfig.GasLimit.Uint64(),
```

Comment:
We probably need this gas limit

Reply:
If the tests are passing without it I don't know why we should specify it?